main_5.cpp

```diff
@@ -0,0 +1,34 @@
+#include "head.cpp"
+#define number_of_stud 3
+#define number_of_marks 3
+
+
+int main(int argc, const char * argv[])
+{
+    student Vasyl;
+    cout<<"greats"<<endl;
+    Vasyl.InputMatrix();
+    cout<<"Vasyl Average score= "<< Vasyl.CalculateTheAverageScore() <<endl<<endl;
+    
+    student Maksym;
+    cout<<"greats"<<endl;
+    Maksym.InputMatrix();
+    cout<<"Maksym Average score= "<< Maksym.CalculateTheAverageScore() <<endl<<endl;
+    
+    student Nika;
+    cout<<"greats"<<endl;
+    Nika.InputMatrix();
+    cout<<"Nika Average score= "<< Nika.CalculateTheAverageScore() <<endl<<endl;
+    
+    groupOFstudents IR_15;
```

Comment:
class name - lowercase?????
and variable name - uppercase??? 

please check naming convention